cordova-lib/spec-plugman/platforms/ios.spec.js

```diff
@@ -301,14 +301,8 @@ describe('ios project handler', function() {
             });
         });
         it('of two plugins should apply xcode file changes from both', function(){
```

Comment:
in your first comment, you had done outside the promise chain. In your second comment, you have done in the correct spot. Order shouldn't matter.